dask_sql/context.py

```diff
@@ -34,6 +34,13 @@
 from dask_sql.physical.rex import RexConverter, core
 from dask_sql.utils import ParsingException
 
+# try:
+#    import dask_cuda  # noqa: F401
+# except ImportError:  # pragma: no cover
+#    pass
+#
```

Comment:
I believe we need to re-add this back prior to jvm startup which happens as a part of line 12.

Reply:
Yep, I had forgotten to add it back after testing that commenting it out actually caused errors